cypress/e2e/registration.cy.js

```diff
@@ -1,11 +1,57 @@
 /// <reference types='cypress' />
 
 describe('Student Registration page', () => {
+  let user;
   before(() => {
-
+    cy.visit('');
+    cy.task('generateUser').then((generateUser) => { user = generateUser; });
   });
-
-  it('', () => {
-
+  it('Registration with all fields except adding picture', () => {
+    cy.get('[placeholder="First Name"]')
+      .type(user.firstName);
+    cy.get('[placeholder="Last Name"]')
+      .type(user.lastName);
+    cy.get('#userEmail')
+      .type(user.email);
+    cy.contains('.custom-control-label', user.gender)
+      .click();
+    cy.get('[placeholder="Mobile Number"]')
+      .type(user.mobile);
+    cy.get('#dateOfBirthInput')
+      .type('{selectall}23 October{enter}');
+    cy.get('.subjects-auto-complete__value-container')
+      .type('en{enter}' + 'phy{enter}');
+    cy.contains('.custom-control-label', user.hobby)
+      .click();
+    cy.get('[placeholder="Current Address"]')
+      .type(user.address);
+    cy.get('#state')
+      .type('{downarrow}{enter}');
+    cy.get('#city')
+      .type('{downarrow}{enter}');
+    cy.get('#submit')
+      .click();
+    cy.get('#example-modal-sizes-title-lg')
+      .should('contain.text', 'Thanks for submitting the form');
+    cy.get('tbody > :nth-child(1) > :nth-child(2)')
+      .should('contain', user.firstName);
```

Comment:
Please use the other selectors. It is not clear exactly what you check with these selectors.
Fix it for this and further checks, please

Reply:
done